eZ/Publish/Core/FieldType/Keyword/KeywordStorage/Gateway/LegacyStorage.php

```diff
@@ -275,9 +280,13 @@ protected function insertKeywords(array $keywordsToInsert, $contentTypeId)
         return $keywordIdMap;
     }
 
-    protected function deleteOldKeywordAssignments($fieldId)
+    protected function deleteOldKeywordAssignments($fieldId, $versionNo)
```

Comment:
`protected function deleteOldKeywordAssignments($fieldId, $versionNo =null)`
to make it tiny bit more clear it's an optional parameter that may be set to null value instead of just int.